include/llfio/v2.0/detail/impl/posix/byte_socket_handle.ipp

```diff
@@ -26,8 +26,12 @@ Distributed under the Boost Software License, Version 1.0.
 #include "import.hpp"
 
 #if LLFIO_EXPERIMENTAL_STATUS_CODE
+#if !OUTCOME_USE_SYSTEM_STATUS_CODE && __has_include("outcome/experimental/status-code/include/status-code/getaddrinfo_code.hpp")
```

Comment:
Is this the right path?

Reply:
Yes, it's identical to the include on L29/L30:
```c++
#include "outcome/experimental/status-code/include/status-code/getaddrinfo_code.hpp"
```

Reply:
It looks like the status code repository isn't cloned by the CI for some reason. But that seems to be the case on `develop` to.